api/clients/v2/payload_disperser.go

```diff
@@ -137,14 +128,12 @@ func (pd *PayloadDisperser) SendPayload(
 	ctx context.Context,
 	certVerifierAddress string,
 	// payload is the raw data to be stored on eigenDA
-	payload []byte,
+	payload *codecs.Payload,
```

Comment:
why is payload defined in `codecs`? Looks a bit weird no? It feels like its something that's purely related to some codec process, whereas payload is super fundamental.

Reply:
Makes sense, I think the `codecs` package containing `Payload`, `Blob`, and `encodedPayload` could just be renamed in its entirety. Any naming suggestions?

`Payload` and `Blob` are both fundamental, and `encodedPayload` feels like it just belongs wherever `Payload` and `Blob` end up.

Reply:
`types`, `datatypes`, `core`?

Reply:
Went with mashup `coretypes` in [650f6800](https://github.com/Layr-Labs/eigenda/pull/1305/commits/650f6800eeafe2fc5105fed0a4432cc016f4f24c), since it seems a good balance of specificity. LMK if you're not a fan

Reply:
sgtm. I guess the only possible issue is that we've been using `core` to mean stuff in eigenda-core, as opposed to integration, and these seem like more integration-like types. But unclear to me if we want this kind of distinction going forward (especially as we bring everything into monorepo). Let's roll with this for now but we can keep thinking about it.